en/advanced-topics/engine-customization.md

```diff
@@ -41,13 +41,17 @@ cd E:/engine
 npm install -g gulp
 # Install dependent modules
 npm install
+# 编译一些数据，包括 debug infos 和 .d.ts
+gulp build
 ```
 
-> **Note**: the gulp build tool is required to generate debuginfos.
+> **Note**: the gulp build tool is required to generate debug infos.
 
 ### 1.4 Make changes and compile
 
-Next, customize the engine modifications and then click **Developer -> Compile the engine** in the Cocos Creator editor menu bar to compile.
+Restarting the editor after the changes are complete recompiles the engine. The first entry takes longer to compile.
+
+Next, if you have changed the engine, you can manually compile it by clicking **Developer -> Build Engine** in the menu bar of the Cocos Creator editor.
```

Comment:
```suggestion
Next, if the engine has been changed, it can be manually compiled by clicking **Developer -> Build Engine** in the menu bar of the Cocos Creator editor.
```